docs/it/schema.rst

```diff
@@ -139,6 +139,8 @@ Questa chiave contiene il logo del software. I loghi dovrebbero essere
 tutti in formato vettoriale; i formati raster sono solo accettabili come
 fallback. In questo caso, dovrebbero essere PNG trasparenti, con una
 larghezza minima di 1000px.
+**Nota bene:** Il file deve risiedere all'interno del repository. 
+Il percorso indicato può essere relativo oppure assoluto. 
```

Comment:
@alranel secondo te può andar bene scritto così? Una volta decisa la forma la riporto in tutte le chiavi interessate.

Reply:
*Questa chiave indica il logo del software. Il valore può essere il percorso relativo al file a partire dalla root del repository, oppure una URL assoluta che punta al logo in versione raw. In entrambi i casi, il file deve risiedere all'interno del medesimo repository che contiene il publiccode.yml.*
*Il logo dovrebbe essere in formato vettoriale [...]*